modules/summary_builder/views/summariser_definition/work_queue.php

```diff
@@ -0,0 +1,93 @@
+<?php
```

Comment:
One for another day maybe - just a thought that it might be useful to have a generic work queue review page (with error details and ability to reset error queue entries).